DATA/production/configurations/asyncReco/async_pass.sh

```diff
@@ -339,32 +339,45 @@ if [[ -n "$ALIEN_JDL_USEGPUS" && $ALIEN_JDL_USEGPUS != 0 ]]; then
   echo "Enabling GPUS"
   export GPUTYPE="HIP"
   export GPUMEMSIZE=$((25 << 30))
-  if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
-    if [[ $keep -eq 0 ]]; then
+  if [[ "ALIEN_JDL_USEFULLNUMADOMAIN" == 0 ]]; then
+    if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
```

Comment:
Could you leave the check for `"0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01"` outside. I use that for my tests, where I set everything externally, and don't want anything to be set by this script. And it is counterintuitive if I have to set `ALIEN_JDL_USEFULLNUMADOMAIN=0` in addition.

Reply:
Please, check the new version.